### PR TITLE
Added connector for Radio Student

### PR DESCRIPTION
**Pull request by dinogrgic1**
**Describe the changes you made**
I added connector for http://www.radiostudent.hr/. This is a student radio in Croatia which embeds songs that were played last with timestamps. Only available information to us is artist, track and timestamp.

**Additional context**
Here is the picture of this element live on website. Website does feature a web player but sadly web player does not contain metadata for scrobbler to function properly so we can only used this latest played element.
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "editor.formatOnSave": true,
 
   "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```
Comment (inverse) [inline .vscode/settings.json:6]: What's the reason for this change?

File: src/connectors/radiostudent.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export {};
+
+Connector.playerSelector = '#icecast_list';
+Connector.artistTrackSelector = '#icecast_list > ul > li > p';
+Connector.currentTimeSelector = '#icecast_list > ul > li > h3';
```
Comment (inverse) [inline src/connectors/radiostudent.ts:5]: This one is incorrect - `currentTimeSelector` is more for like when you have a player like `00:02 / 04:23`, where `00:02` would be the current play time of a track.

**Review by inverse — COMMENTED** [review]
Apologies in the delay in getting back to this one.

I think it should probably work without the `currentTimeSelector`. while it would be super basic in the support it'll provide it'll indeed be better than nothing.

File: src/connectors/radiostudent.ts (hunk 2 above)
Comment (dinogrgic1) [reply src/connectors/radiostudent.ts:5]: Thanks for the clarification! I will remove the `currentTimeSelector`.

File: .vscode/settings.json (hunk 1 above)
Comment (dinogrgic1) [reply .vscode/settings.json:6]: I had issues with Prettier locally, and changed this by mistake. Reverting it.

**Review by inverse — APPROVED** [review]
Thanks for the contribution!
